packages/graphback-runtime-knex/src/PostgreDBDataProvider.ts

```diff
@@ -96,7 +105,8 @@ export class KnexDBDataProvider<Type = any, GraphbackContext = any> implements G
   public async batchRead(relationField: string, ids: string[]): Promise<Type[][]> {
     //TODO: Use mapping when relationships are done
     //tslint:disable-next-line: await-promise
-    const dbResult = await this.db.select().from(this.tableName).whereIn(relationField, ids);
+    // const dbResult = await this.db.select().from(this.tableName).whereIn(relationField, ids);
+    const dbResult = await this.db.query(`SELECT * FROM ${this.tableName} WHERE ${relationField} IN (${ids})`) // this is not the correct implimentation for where clause
```

Comment:
same goes for this...  the ids received is an array of ids

Reply:
Simply check what queries knex is doing in docker-compose logs